Tests/SwiftLintTestHelpers/SwiftLintTestCase.swift

```diff
@@ -0,0 +1,9 @@
+import SwiftLintFramework
+import XCTest
+
+open class SwiftLintTestCase: XCTestCase {
```

Comment:
A lot of the unit test related rules will not fire for custom test case classes, unless they are configured to do so via the `test_parent_classes` parameter, so you may want to update `.swiftlint.yml` appropriately.

Reply:
Thanks, I configured this:

```yaml
single_test_class:
  test_parent_classes:
    - SwiftLintTestCase
    - XCTestCase
```

But there are other unit test rules that should be configured in this same way.

Reply:
YAML anchors to the rescue

```yaml
balanced_xctest_lifecycle: &unit_test_configuration
  test_parent_classes:
    - SwiftLintTestCase
    - XCTestCase
empty_xctest_method: *unit_test_configuration
single_test_class: *unit_test_configuration
```

Reply:
nice - I did not know you could do that.